apps/rule-manager/app/service/RuleManager.scala

```diff
@@ -14,8 +14,21 @@ import db.{DbRuleDraft, DbRuleLive}
 import db.DbRuleDraft.autoSession
 import model.{LTRuleCoreForm, LTRuleXMLForm, RegexRuleForm}
 import play.api.data.FormError
+import play.api.libs.json.Json
 import scalikejdbc.DBSession
 
+object AllRuleData {
```

Comment:
To correctly handle updating rule data client-side post-publication, it made sense to pass back all the rule data – and because we're now doing that in two places, a case class seemed better.